Can't build package with python 3.8

Can't build python3-module-virtualenv-clone with new python version.
Build log:
Cannot build this package.
Please investigate.
Excerpt from build log:

ImportError: No module named ast
_____________________ TestVirtualenvSys.test_clone_version _____________________

self = <tests.test_virtualenv_sys.TestVirtualenvSys testMethod=test_clone_version>

    def test_clone_version(self):
        """Verify version for cloned virtualenvs"""
       for version in start_version_test():

tests/test_virtualenv_sys.py:55: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    def start_version_test():
        ran_once = False
        for version in versions:
            # create a virtualenv
            if subprocess.call(['virtualenv3', '-p', 'python' + version,
                                venv_path]) != 0:
                continue
    
            ran_once = True
            yield version
    
       assert ran_once, "All versions were skipped."
E       AssertionError: All versions were skipped.
E       assert False

tests/test_virtualenv_sys.py:20: AssertionError
----------------------------- Captured stdout call -----------------------------
Running virtualenv with interpreter /usr/bin/python2.7
The path python3.4 (from --python=python3.4) does not exist
The path python3.5 (from --python=python3.5) does not exist
The path python3.6 (from --python=python3.6) does not exist
The path python3.7 (from --python=python3.7) does not exist
----------------------------- Captured stderr call -----------------------------
Traceback (most recent call last):
  File "/usr/lib/python3/site-packages/virtualenv.py", line 20, in <module>
    import ast
ImportError: No module named ast
__________________ TestVirtualenvSys.test_virtualenv_syspath ___________________

self = <tests.test_virtualenv_sys.TestVirtualenvSys testMethod=test_virtualenv_syspath>

    def test_virtualenv_syspath(self):
        """Verify syspath for created virtualenvs"""
      for version in start_version_test():

tests/test_virtualenv_sys.py:44: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    def start_version_test():
        ran_once = False
        for version in versions:
            # create a virtualenv
            if subprocess.call(['virtualenv3', '-p', 'python' + version,
                                venv_path]) != 0:
                continue
    
            ran_once = True
            yield version
    
       assert ran_once, "All versions were skipped."
E       AssertionError: All versions were skipped.
E       assert False

tests/test_virtualenv_sys.py:20: AssertionError
----------------------------- Captured stdout call -----------------------------
Running virtualenv with interpreter /usr/bin/python2.7
The path python3.4 (from --python=python3.4) does not exist
The path python3.5 (from --python=python3.5) does not exist
The path python3.6 (from --python=python3.6) does not exist
The path python3.7 (from --python=python3.7) does not exist
----------------------------- Captured stderr call -----------------------------
Traceback (most recent call last):
  File "/usr/lib/python3/site-packages/virtualenv.py", line 20, in <module>
    import ast
ImportError: No module named ast
__________________ TestVirtualenvSys.test_virtualenv_versions __________________

self = <tests.test_virtualenv_sys.TestVirtualenvSys testMethod=test_virtualenv_versions>

    def test_virtualenv_versions(self):
        """Verify version for created virtualenvs"""
       for version in start_version_test():

tests/test_virtualenv_sys.py:35: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    def start_version_test():
        ran_once = False
        for version in versions:
            # create a virtualenv
            if subprocess.call(['virtualenv3', '-p', 'python' + version,
                                venv_path]) != 0:
                continue
    
            ran_once = True
            yield version
    
       assert ran_once, "All versions were skipped."
E       AssertionError: All versions were skipped.
E       assert False

tests/test_virtualenv_sys.py:20: AssertionError
----------------------------- Captured stdout call -----------------------------
Running virtualenv with interpreter /usr/bin/python2.7
The path python3.4 (from --python=python3.4) does not exist
The path python3.5 (from --python=python3.5) does not exist
The path python3.6 (from --python=python3.6) does not exist
The path python3.7 (from --python=python3.7) does not exist
----------------------------- Captured stderr call -----------------------------
Traceback (most recent call last):
  File "/usr/lib/python3/site-packages/virtualenv.py", line 20, in <module>
    import ast
ImportError: No module named ast
=================== 4 failed, 14 passed in 139.67s (0:02:19) ===================
ERROR: InvocationError for command /usr/src/RPM/BUILD/python3-module-virtualenv-clone-0.5.3/.tox/py38/bin/py.test -v (exited with code 1)
___________________________________ summary ____________________________________
ERROR:   py38: commands failed
error: Bad exit status from /usr/src/tmp/rpm-tmp.27630 (%check)

RPM build errors:
    Bad exit status from /usr/src/tmp/rpm-tmp.27630 (%check)
Command exited with non-zero status 1
138.16user 10.69system 2:42.36elapsed 91%CPU (0avgtext+0avgdata 65796maxresident)k
0inputs+0outputs (0major+2013679minor)pagefaults 0swaps
hsh-rebuild: rebuild of `python3-module-virtualenv-clone-0.5.3-alt1.src.rpm' failed.
Command exited with non-zero status 1

python version: 3.8.1
virtualenv-clone version: 0.5.3

--- a/.travis.yml
+++ b/.travis.yml
@@ -5,6 +5,7 @@ python:
   -   "3.5"
   -   "3.6"
   -   "3.7"
+  -   "3.8"
 dist: xenial
 sudo: true
 

--- a/setup.py
+++ b/setup.py
@@ -55,6 +55,7 @@ setup(name="virtualenv-clone",
         "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
     ],
     tests_require=test_requirements,
     cmdclass={'test': ToxTest}

--- a/tests/__init__.py
+++ b/tests/__init__.py
@@ -8,7 +8,7 @@ from unittest import TestCase
 tmplocation = tempfile.mkdtemp()
 venv_path = os.path.realpath(os.path.join(tmplocation,'srs_venv'))
 clone_path = os.path.realpath(os.path.join(tmplocation,'clone_venv'))
-versions = ['2.7', '3.4', '3.5', '3.6', '3.7']
+versions = ['2.7', '3.4', '3.5', '3.6', '3.7', '3.8']
 
 def clean():
     if os.path.exists(tmplocation): shutil.rmtree(tmplocation)

--- a/tox.ini
+++ b/tox.ini
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py34,py35,py36,py37
+envlist = py27,py34,py35,py36,py37,py38
 
 [testenv]
 commands = py.test -v []

